Issues warning upon calling deprecated generate() function

SaintNazaire opened the pull request.

SaintNazaire committed:
Issues warning upon calling deprecated generate() function

fievelk commented:
@SaintNazaire Even if this code is self explanatory, I would suggest you to provide a description for your future pull requests, in order to make it easier to review them.
Referenced issues: #1180, #736, #1334. 

(I think we could close some of those issues since they all refer to the same problem).

Regarding the content of the PR: I would probably avoid specifying the NLTK version in the warning, or make it dynamic so that it refers the current NLTK version. We do not know if this will be fixed in the next release; if not, we would have to change the warning every time NLTK is updated and I think this is not desirable.

SaintNazaire committed:
Fixing #1180, #736, #1334.

Suggest to close #1180, #736, #1334.
PR updated to incorporate fievelk comments.

stevenbird commented:
I'll merge this and fix the version issue.
